src/test/dependencies.test.ts

```diff
@@ -65,6 +68,47 @@ suite("dependencies", () => {
             listener => myDependency.install("remote", true, listener));
     });
 
+    test("Get tagged asset url", async function() {
+        this.timeout(PRUSTI_UBUNTU_ASSET_DOWNLOAD_TIMEOUT_MS);
+        const assetName = "prusti-release-ubuntu.zip";
+        const url = await GitHubReleaseAsset.getTaggedAssetUrl(
+            "viperproject", "prusti-dev", assetName, "v-2020-10-06-1807");
```

Comment:
I would not rely on `prusti-dev` tags because there is a good chance that I'll delete some of them, as I'm still experimenting.

Reply:
Fixed, I've set up the following repo: https://github.com/viperproject/vs-verification-toolbox-release-testing